data/common/items/items/dog_dollar.lua

```diff
@@ -0,0 +1,59 @@
+local item = Item{
+    -- Item ID (optional, defaults to path)
+    id = "dog_dollar",
+    -- Display name
+    name = "DogDollar",
+
+    -- Item type (item, key, weapon, armor)
+    type = "item",
+    -- Item icon (for equipment)
+    icon = nil,
+
+    -- Battle description
+    effect = "Not\nso\nuseful",
+    -- Shop description
+    shop = nil,
+    -- Menu description
+    description = "A dollar with a certain dog on it.\nIts value decreases each Chapter.",
+
+    -- Shop sell price
+    price = math.ceil(100/Game.chapter),
+
+    -- Consumable target mode (party, enemy, noselect, or none/nil)
+    target = "noselect",
+    -- Where this item can be used (world, battle, all, or none/nil)
+    usable_in = "battle",
+    -- Item this item will get turned into when consumed
+    result_item = nil,
+    -- Will this item be instantly consumed in battles?
+    instant = false,
+
+    -- Equip bonuses (for weapons and armor)
+    bonuses = {
+        attack = 0,
+    },
+    -- Bonus name and icon (displayed in equip menu)
+    bonus_name = nil,
+    bonus_icon = nil,
+
+    -- Equippable characters (default true for armors, false for weapons)
+    can_equip = {},
+
+    -- Character reactions
+    reactions = {},
+}
+
+function item:onWorldUse(target)
+    return false
```

Comment:
this has dialogue in DR according to the code

Reply:
Huh? I checked it in-game and on Deltarune wiki, and it doesn't have one/can't be activated? Maybe it's unused?

Reply:
thats fine then